src/current/v25.4/read-from-standby.md

```diff
@@ -0,0 +1,94 @@
+
+The standby cluster's ReaderVC has its own system tables and cluster settings. The ReaderVC replicates a subset of system tables, including **Users** and **Roles**, from the AppVC, so that existing primary users can authenticate using the same users and roles as on the primary cluster's AppVC. Other system tables and cluster settings are set to defaults in the ReaderVC.
+
+In the event of failover, the ReaderVC is destroyed.
+
+## Use the read from standby feature
+### Before you begin
+
+Prior to setting up read from standby, ensure that:
+
+- you have already configured PCR between a _primary_ cluster and a _standby_ cluster. For information on configuring PCR, refer to [Set Up Physical Cluster Replication]({% link {{ page.version.version }}/set-up-physical-cluster-replication.md %}).
+- your CockroachDB version is v24.3 or later. The `read from standby` option is not supported in earlier versions.
+
+### Start a PCR stream with read from standby
+
+To start a PCR stream that allows read access to the standby cluster, use the `CREATE VIRTUAL CLUSTER ... REPLICATION` statement with the `READ VIRTUAL CLUSTER` option:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE VIRTUAL CLUSTER main FROM REPLICATION OF main ON 'postgresql://{connection string to primary}' WITH READ VIRTUAL CLUSTER;
+~~~
+
+### Add read from standby to a PCR stream
+
+To add read from standby capabilities to an existing PCR stream, use the `ALTER VIRTUAL CLUSTER` statement:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+ALTER VIRTUAL CLUSTER main SET REPLICATION READ VIRTUAL CLUSTER;
+~~~
+
+{{site.data.alerts.callout_info}}
+The standby cluster's AppVC must have a status of `replicating` before you can create your ReaderVC. Use the `SHOW VIRTUAL CLUSTERS` command to check the status of the AppVC.
+{{site.data.alerts.end}}
+
+### Check the status of your reader virtual cluster
+
+To confirm that your reader virtual cluster is active:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SHOW VIRTUAL CLUSTERS;
+~~~
+
+The output shows a `standby-readonly` virtual cluster in addition to the systemVC and AppVC:
+
+~~~
+  id |       name       | data_state  | service_mode
+-----+------------------+-------------+---------------
+   1 | system           | ready       | shared
+   3 | standby          | replicating | none
+   4 | standby-readonly | ready       | shared
+~~~
+
+{{site.data.alerts.callout_info}}
+The ReaderVC cannot serve reads until after the PCR initial scan is complete. After completing the initial scan, wait until the ReaderVC's `service_mode` is `shared`, then wait about one minute before connecting to the ReaderVC.
+{{site.data.alerts.end}}
+
+### Run read-only queries on the standby cluster
+
+Once you have created a reader virtual cluster on the standby cluster, you can connect to it and run read (`SELECT`) queries. For example:
```

Comment:
suggest making `SELECT` a link to https://www.cockroachlabs.com/docs/v25.4/selection-queries.html